src/datetime.js

```diff
@@ -797,15 +797,15 @@ export class DateTime {
    * @type {string}
    */
   get outputCalendar() {
-    return this.loc.outputCalendar;
+    return this.isValid ? this.loc.outputCalendar : null;
   }
 
   /**
    * Get the name of the time zone.
    * @type {string}
    */
   get zoneName() {
-    return this.invalid ? null : this.zone.name;
```

Comment:
This getter also needed to be updated, as in the prototype `invalid` is `undefined`, thus falsy. But as `isValid` explicity checks for `null`, using it works just fine. Using `isValid` also matches the other getters that did not need to be updated.